trip_app/trips/permissions.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework.permissions import IsAuthenticated
+
+
+class IsTripDriverOrAdmin(IsAuthenticated):
+    """
+    Permissions to allow only driver of a trip or admin to edit/delete a trip
```

Comment:
Sentence should be on a first line of docstring like this : `"""Permissions to allow only driver of a trip or admin to edit/delete a trip."""`  and always dot at the end.

Reply:
This is too tough in my opinion. Typically we'd start description from the next line at most of projects. @Meathookseed any specific ideas why the first line is better?

Reply:
https://www.python.org/dev/peps/pep-0257/